spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt

```diff
@@ -144,7 +145,11 @@ public sealed class SparkIcon(@DrawableRes override val drawableId: Int) : IconD
             public object Right : Arrow(R.drawable.spark_icons_chevron_double_right)
         }
 
-        public object Close : Arrows(R.drawable.spark_icons_close)
+        public sealed class Close(@DrawableRes override val drawableId: Int) : Arrows(drawableId) {
+            public object Default : Arrow(R.drawable.spark_icons_close)
```

Comment:
`Default` & `Full` should extend `Close` instead of `Arrows`

Reply:
🙈 good catch!

Reply:
Yes, indeed! I have corrected it in the next commit